src/docs/community/features.mdx

```diff
@@ -1,10 +1,16 @@
-# New Feature Requests
+import { Divider, NewFeatureFormButton } from '../../app/components';
 
-## Features of PX Blue
+# New Feature Requests of PX Blue
 
-If you'd like to see something added to PX Blue, such as new patterns, components, code examples, theme updates, etc., please [Contact us](/community/contactus) or log an issue on [GitHub](https://github.com/pxblue). If you'd like to add a feature yourself, please read our section on [sharing code](/community/sharing) - check out our [Contribution Guidelines](https://github.com/pxblue/.github/blob/master/CONTRIBUTING.md) before you get started.
+If you'd like to see something added to PX Blue, such as new patterns, components, icons, code examples, theme updates, etc., please click the button below and fill up the form.
 
-## New Features in a Framework
+<NewFeatureFormButton />
+
+Alternatively, you may [send us an email](/community/contactus). Developers can also log an issue on [GitHub](https://github.com/pxblue).
```

Comment:
We lost the link out to the innersourcing page. I think it would be nice to keep that here too.